subiquity/curtin.py

```diff
@@ -71,6 +71,11 @@
 # Reworked for flake8, but it does make it harder to read.
 POST_INSTALL_LIST = [
     ("late_commands:"),
+    ("    05_clear_subiquity: curtin in-target -- "
```

Comment:
It's customizable, but we should take the proper steps to clean up anyway. These things *must not* remain on the system.

The core snap is a different matter, and up to discussion (but for now it also wouldn't be included on the installed image).